ugvc/mrd/srsnv_plotting_utils.py

```diff
@@ -244,12 +288,18 @@ def LoD_training_summary(
         ratio_of_bases_in_coverage_range,
         _,
         _,
-    ) = read_effective_coverage_from_sorter_json(cram_stats_file)
+    ) = read_effective_coverage_from_sorter_json(sorter_json_stats_file)
+
+    read_filter_correction_factor = 1
+    if ("X_FILTERED_COUNT" in df_tp) and ("X_READ_COUNT" in df_tp):
+        read_filter_correction_factor = (df_tp["X_FILTERED_COUNT"]).sum() / df_tp["X_READ_COUNT"].sum()
+    else:
+        logger.warning("X_FILTERED_COUNT or X_READ_COUNT no in dataset, read_filter_correction_factor = 1 in LoD")
```

Comment:
Please use the enumeration from featuremap_utils instead of the "X_" strings

Reply:
done